src/stock-receipt/stock-receipt.jsx

```diff
@@ -171,6 +268,105 @@ const StockReceipt = () => {
 								Stock Fetch
 							</Button>
 						</Column>
+						<Column sm={8} lg={4} style={{ paddingTop: "1.5rem" }}>
+							<Button
+								onClick={() =>{setShowModal(true)}}
+								size={"md"}
+								kind="primary"
+								disabled={!isDisabled}
+								className={isDisabled ? styles.buttonColor : ""}
+							>
+								Load Stock
+							</Button>
+							{showModal &&
+							<Modal className="addDrugModal" open={showModal} onRequestClose={handleCloseModal} size='lg' primaryButtonText="Save" secondaryButtonText="Cancel" onRequestSubmit={handleSaveDrugButtonClick}> 
+								    <DataTable
+									rows={rows}
+									headers={["S.No", "Drug Name", "Batch No", "Expiry Date", "Quantity", "Total Quantity", "Actions"]}
+									render={({ rows, headers, getHeaderProps }) => (
+									<TableContainer title="Add New Drug">
+									<Table className={styles.addStocktable}>
+										<TableHead>
+										<TableRow>
+											{headers.map((header, index) => (
+											<TableHeader key={index} {...getHeaderProps({ header })}>
+												{header}
+											</TableHeader>
+											))}
+										</TableRow>
+										</TableHead>
+										<TableBody>
+										{rows.map((row) => (
+											<TableRow key={row.id}>
+											<TableCell>{row.id}</TableCell>
+											<TableCell>  
+												<ComboBox
+													items={dropdownItems}
+													shouldFilterItem={filterItems}
+													selectedItem={row.drugName}
+													onChange={(selectedItem) => handleComboBoxChange(row.id, selectedItem)}
+													style={{ width: "270px" }}
+												/>
+											</TableCell>
+											<TableCell>
+												<TextInput
+												id={`batchNo-${row.id}`}
+												value={row.batchNo}
+												onChange={(e) =>
+													handleInputChange(row.id, "batchNo", e.target.value)
+												}
+												/>
+											</TableCell>
+											<TableCell>
+											<DatePicker
+												datePickerType="single"
+												id={`expiryDate-${row.id}`}
+												dateFormat="d/m/Y"
+												value={row.expiryDate}
+												minDate={currentDate}
+												onChange={(date) => handleInputChange(row.id, 'expiryDate', date[0])}
+											>
+												<DatePickerInput
+												value={row.expiryDate}
+												onChange={(e) => handleInputChange(row.id, 'expiryDate', e.target.value)}
+												pattern="(0[1-9]|[1-2][0-9]|3[0-1])/(0[1-9]|1[0-2])/\\d{4}"
```

Comment:
Since we already have a date-utils.js file can we move this pattern to that file and import here so that we can reuse it if required

Reply:
Done

Reply:
done